### DISCUSSION TITLE
Seeking example of password recovery flow using @supabase/ssr

### DISCUSSION THREAD

**Discussion by austingebauer** [supabase/supabase#20145] [Questions]
I'm seeking a working example of the password reset flow using [@supabase/ssr](https://github.com/supabase/auth-helpers/tree/main/packages/ssr). I've tried various approaches without success, starting with following supabase [documentation](https://supabase.com/docs/guides/auth/passwords#resetting-a-users-password-forgot-password). I've seen a number of different errors depending on my approach. At this point, I would just like someone to show me a simple working example or documentation that will lead me to that. I'm using Next v14.0.4 and Supabase 1.127.4. 

Thanks!

**Comment by austingebauer** [comment]
Is there anything that I can provide to make answering this question easier? I _really_ like the supabase platform. It's a bummer that essential bits of implementing authentication aren't clearly documented or working as documented. I'd really appreciate some help.

**Comment by charislam** [comment]
Hi @austingebauer, it might help if you have some sample code of what you've tried, or maybe some examples of errors you've seen? What was it about the code examples from that page that didn't work for you?

**Comment by austingebauer** [comment]
@charislam - Thanks for the response! I will throw together an example in a Github repo today and get back to you with a link.

**Comment by austingebauer** [comment]
I was able to get this sorted out when putting the example together. The root of my issue was https://github.com/vercel/next.js/issues/48230. Thanks for offering to help @charislam!

**Reply by taiwonaf** [reply]
@austingebauer can you please share the password reset flow. I am also currently struggling with this. Thank you

**Comment by charislam** [comment]
Glad you got it sorted out!
